index.html

```diff
@@ -0,0 +1,37 @@
+<html>
+  <h2>Welcome to Codius!</h2>
```

Comment:
I wonder if this message should be addressed more to users of the host that the operator?
Example:

```html
<h2>Codius Host</h2>
<p>This is server is a Codius Host running <a href="https://github.com/codius/codiusd">codiusd</a>. You can deploy contracts to this host using the <a href="https://github.com/codius/codius">Codius Client Command Line Interface (CLI)</a>.</p>
<p>For more information on codius visit <a href="https://codius.org">codius.org</a></p>
```